cumulusci/core/keychain/base_project_keychain.py

```diff
@@ -170,7 +176,16 @@ def _load_services(self):
         pass
 
     def _load_default_services(self):
-        pass
+        self._default_services["connected_app"] = DEFAULT_CONNECTED_APP_NAME
+
+    def _load_default_connected_app(self):
+        """Load the default connected app as a first class service on the keychain.
+        Sets this as the default connected app service."""
+        if "connected_app" not in self.config["services"]:
+            self.config["services"]["connected_app"] = {}
+        self.config["services"]["connected_app"][
+            DEFAULT_CONNECTED_APP_NAME
+        ] = DEFAULT_CONNECTED_APP
```

Comment:
The docstring says this sets it as the default, but the code is doing it in a way that won't do that, isn't it?

Reply:
You're correct. This should only load the service.